### ISSUE TITLE
URL text color not blue

### ISSUE THREAD

**Issue by michmuel** [openoereb/pyramid_oereb_mfp#98]
URL text color should be blue (see PDF page 3). Compare specs appendix 1 page 2.

PDF

[CH871277778361_comments.pdf](https://github.com/openoereb/pyramid_oereb_mfp/files/8349536/CH871277778361_comments.pdf)

Specs

https://www.cadastre.ch/content/cadastre-internet/de/manual-oereb/service/extract/static/_jcr_content/contentPar/tabs_copy_copy_copy_/items/dokumente/tabPar/downloadlist/downloadItems/426_1475413822017.download/Weisung-statischer-Auszug-de.pdf

**Comment by marionb** [comment]
![image](https://user-images.githubusercontent.com/1460598/165732447-097ed73a-65ca-428b-ab2b-e54d148536f6.png)

This concerns the text at the end of the paragraph: `www.cadastre.ch` which should be an url.

How important is this? It will not be easy to sole this because this is a text that is provided and the URL is not automatically seen as a URL but simple text. So this can not be solved withing MFP. We would have to change the way the text is passed to MFP in pyramid_oereb or even further up the text is given by the federation.

**Comment by voisardf** [comment]
Very, very low priority for this URL color. We might want to try to solve this, when all other issues are solved. I cant explain the problem to Swisstopo if needed.

**Comment by michmuel** [comment]
I agree with @voisardf. It would certainly improve the appearance of the PDF but it is not worth to invest much time.
